src/Twig/RecordExtension.php

```diff
@@ -75,8 +75,12 @@ public function dummy_with_env(Environment $env, $input = null)
         return $input;
     }
 
-    public function pager(Environment $env, Pagerfanta $records, string $template = '_sub_pager.twig', string $class = 'pagination', string $theme = 'default', int $surround = 3)
+    public function pager(Environment $env, ?Pagerfanta $records, string $template = '_sub_pager.twig', string $class = 'pagination', string $theme = 'default', int $surround = 3)
```

Comment:
how can $records be null instead of empty array / empty pagerfanta object?

Reply:
People invoking `{{ pager(records) }}` when records is null. For instance on an empty search page.

Reply:
on empty search page, `records` should be an empty iterable, not null